config/samples/simplekafkacluster.yaml

```diff
@@ -13,6 +13,10 @@ spec:
   clusterImage: "wurstmeister/kafka:2.12-2.3.0"
   readOnlyConfig: |
     auto.create.topics.enable=false
+  rackAwareness:
```

Comment:
I think we should not add rack awareness config to the simplecluster setup.

Reply:
Nice catch